examples/pytorch/README.md

```diff
@@ -1,106 +1,64 @@
-# Record GPU usage
-
-While running a Python process, open **another terminal** and run:
-
-    bash all_scripts/memory_record_moreh.sh ${DEVICE_ID} | tee ${LOG_FILE}
-
-- DEVICE_ID is the id of the Moreh device you are running the python process on and whose memory you want to record
-
-- LOG_FILE is the file you want to save the memory to
-
-For example, when I have 2 different processes like this
-
-![Process](image.png)
-
-If I want to record the memory of device 1, I will run
-
-    bash all_scripts/memory_record_moreh.sh 1 | tee memory.log
-
 # Train individual models
 
-## Script
-
-Inside each task's folder, there's a bash script for training:
-
-This script takes into 4 arguments
+Inside each task's folder, there can be 1 or multiple training .sh scripts.
 
-- `model`: name of the model to be downloaded from huggingface hub
-- `batch_size`
-- `output_dir`: where to save checkpoints and results
-- `gpu_model`: which gpu model to run
+Basic arguments:
 
-And then it runs the Python training command.
-
-Modify this script if you want to change other arguments to the Python training file.
-These scripts take more than 4 agruments:
-- `text-classifiaction/train_glue.sh`: take an arguments `task_name` in one of this list: cola, sst2, mrpc, stsb, qqp, mnli, qnli, rte, wnli.
-- - Examples: `bash train_glue.sh bert-base-uncased 64 /output mrpc`
-
-## Run
-
-Usage:
-
-    cd ${task}
-    bash ${train_script} ${model} ${batch_size} ${output_dir}
+- `-m model` : name of the model to be downloaded from huggingface hub
+- `-b batch_size`
+- `-g gpu` : id of target gpu on your Moreh machine
 
 Example:
 
     cd multiple-choice
-    bash train.sh google/mobilebert-uncased 64 outputs
-
-## Multiple training scripts
+    bash train.sh -m google/mobilebert-uncased -b 64 -g 1
```

Comment:
Are all the flags (-b 64 -g 1 -m) optional or necessary here? Ideally they should be optional as the users don't have to figure out what values of the flags to run the models.